spec/presenters/guide_presenter_spec.rb

```diff
@@ -28,15 +28,18 @@
       expect(presenter.content_payload).to be_valid_against_schema('service_manual_guide')
     end
 
+    describe "common service manual draft payload" do
+      let(:payload) { presenter.content_payload }
+
+      include_examples "common service manual draft payload"
```

Comment:
Unless I'm missing a subtle difference between `include_examples` and `it_behaves_like "[a…]"` I find the latter works a lot better within the 'english as code' context.

But that's probably also just because I'm more used to it, happy to be persuaded otherwise.